dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -2395,15 +2393,25 @@ private void makeFetchObject(StringBuilder sb)
         }
     }
 
-    private String makeGetVersionRequest()
+    private String makeGetVersionRequest(Span versionSpan)
     {
+        String traceContextHeader = null;
```

Comment:
I believe getTraceParentHeader will return null if span == null, or invalid. It feels a little cleaner if we change all these call to gettraceparentheader to:

String traceContextHeader = org.hpccsystems.ws.client.utils.Utils.getTraceParentHeader(versionSpan);